delira/_backends.py

```diff
@@ -83,3 +83,37 @@ def get_backends():
     if not __BACKENDS:
         _determine_backends()
     return __BACKENDS
+
+
+def seed_all(seed):
+    """
+    Helper Function to seed all available backends
+
+    Parameters
+    ----------
+    seed : int
+        the new random seed
+
+    """
+    import sys
+
+    import numpy as np
+    np.random.seed(seed)
+
+    import random
+    random.seed = seed
+
+    if "torch" in sys.modules and "TORCH" in get_backends():
+        import torch
+        torch.random.manual_seed(seed)
```

Comment:
https://pytorch.org/docs/stable/notes/randomness.html uses `torch.manual_seed`, is there a difference?

Reply:
No it isn't: https://github.com/pytorch/pytorch/blob/master/torch/__init__.py#L180